src/main/python/image_processing/thresholding_manager.py

```diff
@@ -5,12 +5,29 @@ class ThresholdingManager:
     """
     The Thresholding manager is responsible for applying the different types of thresholding techniques
     """
-    def __init__(self):
+    def __init__(self, type):
         """
         Initialise the Thresholding manager
         """
+        self.type = type
         print("Initialise Thresholding Manager")
 
+    def apply(self, image):
+        """
+        This performs the thresholding based on the predefined technique
+        Author(s):
+            Nicolai van Niekerk
+        Args:
+            image: The image to which the thresholding must be applied
+        Returns:
+            obj:'OpenCV image': The thresholded image
+        Todo:
+        """
+        if self.type == "adaptive":
+            return self.adaptiveThresholding(image)
+        elif self.type == "otsu":
```

Comment:
Instead of having only `elif` statements, add an `else` clause to use the default value for in case someone makes a spelling mistake somewhere. Also add a print statement to show which type of threshold manager we are using so that we can see it in the server.